_stbt/core.py

```diff
@@ -1037,14 +1037,6 @@ def ocr(self, frame=None, region=Region.ALL,
         if frame is None:
             frame = self._display.pull_frame()
 
-        if region is None:
-            warnings.warn(
-                "Passing region=None to ocr is deprecated since 0.21 and the "
-                "meaning will change in a future version.  To OCR an entire "
-                "video frame pass region=Region.ALL instead",
-                DeprecationWarning, stacklevel=2)
-            region = Region.ALL
```

Comment:
I wonder if we should have gone through a transition where this `raise`d an exception.  There may be people out there who have just not been seeing this warning.